src/modules/85-cv/pages/health-source/common/MetricThresholds/MetricThresholds.utils.ts

```diff
@@ -172,6 +172,82 @@ export function getIsShowLessThan(
   return false
 }
 
+function getIsAllRequiredValuesPresent(thresholdValueToCompare: MetricThresholdType): boolean {
+  return [
+    thresholdValueToCompare.metricType,
+    thresholdValueToCompare.metricName,
+    thresholdValueToCompare.spec.action,
+    thresholdValueToCompare.criteria.type
+  ].every(value => value !== undefined)
```

Comment:
why only check  undefined

```suggestion
  ].every(value => Boolean(value) )
```

Reply:
this can be empty string also, so I kept it in this way

Reply:
Updated